earth_enterprise/rpms/build.gradle

```diff
@@ -43,20 +43,6 @@ apply plugin: "nebula.ospackage"
 
 def rpmPlatformString = com.tsciences.os.Platform.rpmPlatformString
```

Comment:
do we want 'tsciences' to be part of the platform string ?

Reply:
I think that's being addressed in a different PR

Reply:
Hm. Maybe you'd want to merge some commits before the PR with these changes gets squashed and merged into `master`.  Or, you can resolve merge conflicts afterwards.

Reply:
Depending on timing I can either resolve the conflicts with master on this PR or with the merge from postrgres_upgrade to master. I think it makes sense to wait until the other PR is merged before trying to address those issues.